core/src/zxing/DecodeHints.cpp

```diff
@@ -29,7 +29,7 @@ using zxing::DecodeHints;
 // VC++
 using zxing::BarcodeFormat;
 
-const DecodeHintType DecodeHints::CHARACTER_SET;
+const DecodeHintType DecodeHints::CHARACTER_SET = 1 << 30;
```

Comment:
To be clear, I mean don't include this line at all. It's already declared and defined in the header.